lib/common/messages.c

```diff
@@ -219,7 +219,7 @@ pcmk__message_name(const char *name)
  *       g_hash_table_destroy().
  */
 GHashTable *
-pcmk__register_handlers(pcmk__server_command_t *handlers)
+pcmk__register_handlers(const pcmk__server_command_t handlers[])
```

Comment:
Yep, glib doesn't provide gconstpointer versions of the hash table functions, so everything gets casted to gpointer. From the language point of view, alternate versions of g_hash_table_insert() for each combination of constant or variable key and value would be ideal, but from the human point of view that would be awkward :)

As an aside, const is usable here because we don't specify destroy functions for the key/value (i.e. we use g_hash_table_new() rather than g_hash_table_new_full()).